### PR TITLE
Feature/exam notifications

### PR DESCRIPTION
**Pull request by OgnjenGligoric**

### CODE REVIEW

File: Konteh/back-office-app/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ const routes: Routes = [
   { path: 'edit-question/:id', component: CreateQuestionComponent, canActivate: [AuthGuard] },
   { path: 'exams-overview', component: ExamsOverviewComponent, canActivate: [AuthGuard] },
   { path: 'not-found', component: NotFoundComponent },
+  { path: 'notifications', component: ExamNotificationsComponent },
```
Comment (Batranovic) [inline Konteh/back-office-app/src/app/app-routing.module.ts:18]: Maybe add AuthGuard if only admins can see this page

File: Konteh/back-office-app/src/app/features/exams/exam-notifications/exam-notifications.component.ts [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
 export class ExamNotificationsComponent implements OnInit {
-  receivedMessage: string = "test";
+  receivedMessage: GetAllExamsResponse | null = null;
 
   constructor(private notificationsService: NotificationsService) {}
 
   ngOnInit(): void {
     this.notificationsService.messageReceived.subscribe((message) => {
       this.receivedMessage = message;
+      console.log(this.receivedMessage);
```
Comment (Batranovic) [inline Konteh/back-office-app/src/app/features/exams/exam-notifications/exam-notifications.component.ts:17]: You should delete this log if it was used for testing purposes

**Review by Batranovic — COMMENTED** [review]
Consider my review

File: Konteh/back-office-app/src/app/features/exams/exam-notifications/exam-notifications.component.ts (hunk 2 above)
Comment (OgnjenGligoric) [reply Konteh/back-office-app/src/app/features/exams/exam-notifications/exam-notifications.component.ts]: The whole component was for testing. I am deleting it now.

**Review by laslorajda — APPROVED** [review]

**Pull request merged**